resources/resources_seqr_qc.py

```diff
@@ -28,9 +28,9 @@ def callset_vcf_path(
     Returns callset vcf path. Can be internal or external, exomes or genomes.
     """
     vcf_file_string = (
-        f"sharded_vcf/RDG_{data_type}_Broad_{data_source}.filtered.*.vcf.gz"
+        f"sharded_vcf/*.bgz"
         if sharded
-        else f"RDG_{data_type}_Broad_{data_source}.vcf.bgz"
+        else f"*.vcf.bgz"
```

Comment:
Turns out this wildcard is breaking for our WES - (.bgz) vs WGS (.vcf.gz)

L33 should probably be `else f"*.vcf.gz"`